exercises/concept/pacman-rules/test/Tests.hs

```diff
@@ -0,0 +1,44 @@
+import PacmanRules (eatsGhost, loses, scores, wins)
+import Test.Hspec (describe, hspec, it, shouldBe)
+
+main :: IO ()
+main = hspec $ do
+  describe "eatsGhost" $ do
+    it "ghost eaten when touched while powered" $ do
+      (eatsGhost True True) `shouldBe` True
+
+    it "ghost not eaten when not touched" $ do
+      (eatsGhost True False) `shouldBe` False
+
+    it "ghost not eaten when touched un-powered" $ do
+      (eatsGhost False True) `shouldBe` False
+
+  describe "scores" $ do
+    it "when eating dot" $ do
+      (scores False True) `shouldBe` True
+
+    it "when eating power pellet" $ do
+      (scores True False) `shouldBe` True
+
+    it "nothing when nothing eaten" $ do
+      (scores False False) `shouldBe` False
+
+  describe "loses" $ do
+    it "if touching ghost un-powered" $ do
+      (loses False True) `shouldBe` True
+
+    it "is False if touching ghost while powered" $ do
+      (loses True True) `shouldBe` False
+
+    it "is False if not touching a ghost" $ do
+      (loses True False) `shouldBe` False
+
+  describe "wins" $ do
+    it "if all dots are eaten" $ do
```

Comment:
It did originally, I opted to change it because the test output flows better this way 
e.g. 
```
wins
    if all dots are eaten
```
instead of 
```
wins
   is True if all dots are eaten
```
I can change it back if the consistency in the tests code is preferred.

With the False cases I didn't see a way to make it less verbose.